composer.json

```diff
@@ -6,24 +6,24 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ~9.0.0",
```

Comment:
This is needed due to the following issue: https://www.drupal.org/project/remote_stream_wrapper/issues/3185015
The patch works for 9.1 but it would fail on lower versions.